### PR TITLE
[5.3] Command to create notifications table

### PR DESCRIPTION
**Pull request by mattmcdev**

### CODE REVIEW

**Comment by taylorotwell** [comment]
The migration stub doesn't match the migration doc blocks and such created by the framework normally.

**Comment by mattmcdonald-uk** [comment]
@taylorotwell I think that's sorted now

File: src/Illuminate/Notifications/Console/stubs/notifications.stub [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('notifications', function (Blueprint $table) {
+            $table->string('id')->primary();
+            $table->string('type');
+            $table->string('notifiable_type');
+            $table->integer('notifiable_id');
```
Comment (JosephSilber) [inline src/Illuminate/Notifications/Console/stubs/notifications.stub]: You can use `$table->morphs('notifiable')`. It'll even create the index.

Comment (mattmcdonald-uk) [inline src/Illuminate/Notifications/Console/stubs/notifications.stub]: I had originally, but I presumed that was what @taylorotwell was referring to with it not matching doc blocks as the documentation for notifications doesn't use the morphs shorthand.

**Pull request merged**